### DISCUSSION TITLE
How to save best checkpoint?

### DISCUSSION THREAD

**Discussion by sarmientoj24** [open-mmlab/mmdetection#9165] [Q&A] [answered]
I cannot figure out from the documentation how to

**Comment by jbwang1997** [comment]
You can set `save_best='mAP'` in evaluation in config.

**Comment by sarmientoj24** [comment]
where in evaluation config?

**Comment by jbwang1997** [comment, marked as answer]
It normally is at configs/_base_/datasets/coco_detection.py you can override it in your local config. Can refer to 
https://github.com/open-mmlab/mmdetection/blob/1376e77e6ecbaad609f6003725158de24ed42e84/configs/yolox/yolox_s_8x8_300e_coco.py#L151
